sqldb/sqlc/migrations/000001_invoices.up.sql

```diff
@@ -129,26 +153,8 @@ CREATE TABLE IF NOT EXISTS invoice_htlc_custom_records (
     value BLOB NOT NULL,
 
     -- The htlc id this record belongs to. 
-    htlc_id BIGINT NOT NULL REFERENCES invoice_htlcs(id)
+    htlc_id BIGINT NOT NULL REFERENCES invoice_htlcs(id) ON DELETE CASCADE
 );
 
 CREATE INDEX IF NOT EXISTS invoice_htlc_custom_records_htlc_id_idx ON invoice_htlc_custom_records(htlc_id);
 
--- invoice_payments contains the information of a settled invoice payment. 
-CREATE TABLE IF NOT EXISTS invoice_payments (
-    -- The id for this invoice payment. Translates to SettleIndex.
-    id INTEGER PRIMARY KEY,
-    
-    -- When the payment was settled.
-    settled_at TIMESTAMP NOT NULL,
-
-    -- The amount of the payment in millisatoshis. This is the sum of all the 
-    -- the htlcs settled for this payment.
-    amount_paid_msat BIGINT NOT NULL,
-
-    -- The invoice id this payment is for.
-    invoice_id INTEGER NOT NULL REFERENCES invoices(id)
-);
-
-CREATE INDEX IF NOT EXISTS invoice_payments_settled_at_idx ON invoice_payments(settled_at);
```

Comment:
We should also remove them in the downgrade migration file. And same for other deleted tables and indexes.